.github/workflows/windows-periodic.yml

```diff
@@ -0,0 +1,169 @@
+# Workflow intended to periodically test containerd integration tests
+
+name: win-periodic-integration
+
+on:
+  schedule:
+  - cron: "0 */12 * * *"
```

Comment:
Daily should be fine for a job like this, `0 0 * * *` to run at midnight, or maybe `0 1 * * *` to run after the nightly build jobs